Documentation/Global/Be/_Infobox/_ReferenceIndex.html

```diff
@@ -0,0 +1,18 @@
+<f:be.infobox
+        title="{f:translate(key:'LLL:EXT:lowlevel/Resources/Private/Language/locallang.xlf:checkScript_headline')}"
```

Comment:
And 8 spaces again. 4 or 6 should be OK

Reply:
We had not .html files in the editorconfig, I copied the one from the core over